examples/guestbook/README.md

```diff
@@ -63,18 +68,58 @@ The web front end interacts with the redis master via javascript redis API calls
 
-### Step Zero: Prerequisites
+### Prerequisites
+
+This example requires a running Kubernetes cluster. See the [Getting Started guides](../../docs/getting-started-guides/) for how to get started. And follow the [Prerequisites](../../docs/user-guide/prereqs.md) to make sure your `kubectl` is ok. As noted above, if you have a Google Container Engine cluster set up, go [here](https://cloud.google.com/container-engine/docs/tutorials/guestbook) instead.
+
+### Quick Start
+
+This section shows a simplest way to get the example work. If you want to know the details, you should pass this and read the rest of the example.
+
+Start the guestbook with one command:
+
+```console
+$ kubectl create -f https://raw.githubusercontent.com/kubernetes/kubernetes/master/examples/guestbook/all-in-one/guestbook-all-in-one.yaml
+service "redis-master" created
+replicationcontroller "redis-master" created
+service "redis-slave" created
+replicationcontroller "redis-slave" created
+service "frontend" created
+replicationcontroller "frontend" created
+```
+
+You also can start the guestbook by running:
+
+```console
+$ kubectl create -f examples/guestbook/
+```
+
+Then, list all your services:
+
+```console
+$ kubectl get services
+NAME              CLUSTER_IP       EXTERNAL_IP       PORT(S)       SELECTOR                            AGE
+frontend          10.0.93.211      <none>            80/TCP        app=guestbook,tier=frontend         1h
+redis-master      10.0.136.3       <none>            6379/TCP      app=redis,role=master,tier=backend  1h
+redis-slave       10.0.21.92       <none>            6379/TCP      app=redis,role=slave,tier=backend   1h
+```
+
+And you can access the guestbook on each node with frontend service's `<ClusterIP>:Port`, as it's `10.0.93.211:80` in this guide. As the `<ClusterIP>` is an a cluster-internal IP, if you want to access the guestbook from outside of the cluster, you can add `type: NodePort` to frontend service `spec` field. Then you can access the guestbook with `<NodeIP>:NodePort` from outside of the cluster. There are other ways for you to access the guestbook. You may learn from [Accessing services running on the cluster](../../docs/user-guide/accessing-the-cluster.md#accessing-services-running-on-the-cluster).
 
-This example requires a running Kubernetes cluster.  See the [Getting Started guides](../../docs/getting-started-guides/) for how to get started. As noted above, if you have a Google Container Engine cluster set up, go [here](https://cloud.google.com/container-engine/docs/tutorials/guestbook) instead.
+Before continuing to the gory details, we also recommend you to read [Quick walkthrough](../../docs/user-guide/#quick-walkthrough), [Thorough walkthough](../../docs/user-guide/#thorough-walkthrough) and [Concept guide](../../docs/user-guide/#concept-guide).
```

Comment:
What's the purpose of quick-start? Do users read either quick-start or details, or do users read both? This paragraph assumes that the users read both, but it'd be weird to have created those resources and continue reading the rest of the examples. Plus, if the users only read quick-start, we should at least tell them how to clean up.